ridemyway/__init__.py

```diff
@@ -48,6 +49,13 @@ def create_app(config_name):
     app.database = DATABASE
     app.config.from_object(config[config_name])
     app.conn = app.config['DB_CONN']
+    app.jwt = JWTManager(app)
+    app.blacklist = set()
+
+    @app.jwt.token_in_blacklist_loader
+    def check_if_token_in_blacklist(decrypted_token):
+        jti = decrypted_token['jti']
```

Comment:
What does jti stand for?

Reply:
The Identity of the token being checked if it's in the blacklist. _json token identity_ that is JWT ID